td-shim-tools/src/tee_info_hash.rs

```diff
@@ -32,6 +33,25 @@ const MEM_PAGE_ADD_GPA_SIZE: usize = 0x8;
 const MR_EXTEND_ASCII_SIZE: usize = 0x9;
 const MR_EXTEND_GPA_OFFSET: usize = 0x10;
 const MR_EXTEND_GPA_SIZE: usize = 0x8;
+const OVMF_TABLE_FOOTER_GUID_OFFSET: usize = 0x30;
+
+const OVMF_TABLE_FOOTER_GUID: Guid = Guid::from_fields(
+    0x96b582de,
```

Comment:
Nit: this would be clearer if you added an underscore:

```suggestion
    0x96b5_82de,
```